app/src/test/kotlin/org/imperial/mrc/modelserver/unit/templates/LoginTests.kt

```diff
@@ -0,0 +1,54 @@
+package org.imperial.mrc.modelserver.unit.templates
+
+import org.junit.jupiter.api.Test
+import org.springframework.ui.ConcurrentModel
+import org.assertj.core.api.Assertions.assertThat
+
+class LoginTests
```

Comment:
Rules have been deprecated in JUnit5 and replaced with extensions, but since we weren't really using the Freemarker Template Rule as much as a rule since it lives in the companion object of the test class, I've just converted it into an ordinary class.